verifier/helpers.go

```diff
@@ -127,12 +128,15 @@ func getVerificationPluginMinVersion(signerInfo *signature.SignerInfo) (string,
 	if err != nil {
 		return "", err
 	}
-	// empty version
 	if strings.TrimSpace(version) == "" {
 		return "", fmt.Errorf("%v from extended attribute is an empty string", HeaderVerificationPluginMinVersion)
 	}
-	if !semVerRegEx.MatchString(version) {
+	if isVersionSemverValid(version) {
 		return "", fmt.Errorf("%v from extended attribute is not a valid SemVer", HeaderVerificationPluginMinVersion)
 	}
 	return version, nil
 }
+
+func isVersionSemverValid(version string) bool {
+	return !semVerRegEx.MatchString(version)
```

Comment:
As per name of method `isVersionSemverValid` the method should return true if given version is valid semver.
```suggestion
	return semVerRegEx.MatchString(version)
```

Reply:
Modified accordingly